### PR TITLE
BETA: Register tgnick.is-a.dev

### PR DESCRIPTION
**Pull request by itzkera**
Added `tgnick.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/echo.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TGNick",
+        "email": "drnickmcstuffin@gmail.com"
+    },
+    "record": {
+        "CNAME": "192.0.2"
```
Comment (Stef-00012) [inline domains/echo.json:7]: Invalid CNAME record (must be a domain)

File: domains/tgnick.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TGNick",
+        "email": "drnickmcstuffin@gmail.com"
+    },
+    "record": {
+        "URL": "discord.com"
```
Comment (Stef-00012) [inline domains/tgnick.json:7]: invalid URL record, missing protocol

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/echo.json (hunk 1 above)
Comment (TGNick) [reply domains/echo.json:7]: Oh

File: domains/tgnick.json (hunk 2 above)
Comment (TGNick) [reply domains/tgnick.json:7]: Ok

File: domains/echo.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TGNick",
+        "email": "echoogfn@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://echomp.github.io/Echodownload.github.io/"
```
Comment (Stef-00012) [inline domains/echo.json:7]: Invalid CNAME record
can nout contain `https://` or `/`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**